web_app/src/store/trips.ts

```diff
@@ -0,0 +1,46 @@
+import {action, thunk} from "easy-peasy"
+import {Trip, Trips} from "../model/trip"
+import firebase from '../firebase-config'
+
+const trips: Trips = {
+  items: [],
+  setTrips: action((state, payload) => {
+    state.items = payload
+  }),
+  loading: false,
+  error: false,
+  setError: action((state, payload) => {
+    state.error = payload
+  }),
+  setLoading: action((state, payload) => {
+    state.loading = payload
+  }),
+  fetch: thunk(async actions => {
```

Comment:
Til en annen gang:

`easy-peasy` har en enkel mekanisme for dependency injection. Det kan sikkert være lurt å lage et slags repository med firebase-"kall", også injecte de her. Da blir det ihvertfall mulig å teste, hvis vi noen gang gidder det ;)

https://easy-peasy.now.sh/docs/typescript-tutorial/using-typed-injections.html